pkg/kubelet/cm/container_manager_linux_test.go

```diff
@@ -268,3 +268,35 @@ func TestGetCapacity(t *testing.T) {
 		})
 	}
 }
+func TestNewSystemCgroups(t *testing.T) {
+	cases := []struct {
+		name   string
+		cmName string
+	}{
+		// Not sure what tests should fail here
+		{
+			name:   "Create cgroup container manager with name 'kube'",
+			cmName: "kube",
+		},
+		{
+			name:   "Create cgroup container manager with name '1234'",
+			cmName: "1234",
+		},
+		// I think this should fail but there is no check for system container name
```

Comment:
There looks to be a check for the name [here](https://github.com/JayKayy/kubernetes/blob/99432c1baba90de09c4af6130541483876d05755/pkg/kubelet/cm/container_manager_linux.go#L478) and [here](https://github.com/JayKayy/kubernetes/blob/99432c1baba90de09c4af6130541483876d05755/pkg/kubelet/cm/container_manager_linux.go#L492). This function won't be called unless the container name is valid/non-empty. I don't think this case is needed here.